AreaFiftyLAN/lancie-frontend
# Update imports

These updates should result in significant performance increases for the first loading time of the homepage. For this, all elements import their respective dependencies.

Since the build was broken, I had to remove all sub-modules and update the `gulpfile.js`.

Lastly, there are some small styling fixes to prevent flickering on very slow (regular 3G) networks as well as lazily loading the webcomponents polyfill.

# How to test this branch?

1. Checkout the branch
* Fix the git submodules. Presumably it should work on checkout. If not, remove all directories and `.gitmodules` and then `git reset --hard` while on the branch
* Move `bower_components` to `app/bower_components`
* Run `bower update`
* Run `npm install` and `npm update`
* Run `gulp serve:dist`
* Open the website in your browser (preferably Chrome)
* Enable throttling to various throttling levels (regular 4G, good 3G and regular 3G)
